### ISSUE TITLE
Animation import fails

### ISSUE THREAD

**Issue by 1702707** [BadDogSkyrim/PyNifly#180]
I've been trying to import animations from Skyrim LE and get this error:
![Error](https://github.com/BadDogSkyrim/PyNifly/assets/57536471/05cb199d-9dc7-4a7e-a5d0-a2b1500df795)
I've tried version 12.4, 12.0 and 11.0 and get the same issue which probably means that I'm the one thats done something wrong but I don't know what.
Nif skeleton and meshes import fine.
Using blender 3.0

**Comment by BadDogSkyrim** [comment]
Is this animation the same as 1hm_1stp_walk.hkx?

**Comment by 1702707** [comment]
Yes, I am trying to import 1hm_1stp_walk.hkx, but I get the same error with any animation

**Comment by podvornyakva** [comment]
Hey, there. What is necessary to install for animation import? Havox? Console say that "can't create *.xml" in "/users/steamuser/Temp/"
